src/tools/kata-ctl/README.md

```diff
@@ -27,6 +27,11 @@ $ make
 $ make install
 ```
 
+If you would like to install the tool to a specific directory, then you can provide it through the `INSTALL_PATH` variable.
+```bash
+$ make install INSTALL_PATH=<install_path>
```

Comment:
This line cause a static check error: `ERROR: shell code in file 'src/tools/kata-ctl/README.md' is not valid`.
Maybe use a real example will pass:

```
$ make install INSTALL_PATH=/opt/bin
```